### PR TITLE
move index.ios.js to index.js

### PR DESCRIPTION
**Pull request by kmsolberg**
Beginning to rearrange file structure for client-side folders

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+
+export default class moustache extends Component {
+    render() {
+        return (
+            <View>
+                <Text>
+                    Welcome to Green Moustache!!
+          </Text>
```
Comment (10thfloor) [inline js/index.js]: Please fix your indentation before submitting.

Comment (kmsolberg) [reply js/index.js]: Fixed indentation!

**Comment by 10thfloor** [comment]
What Issue/task is this related to?

**Comment by wtLau** [comment]
finished folder structure

**Pull request merged**